react/wbooks/src/app/components/Login/styles.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../../scss/colors';
+
+@import '../../../scss/components';
+
+.container {
+  border-top: 8px solid $cerulean;
+  margin: auto;
+  max-width: 300px;
+  padding: 24px;
+  width: 100%;
+}
+
+.error {
+  color: $red;
+}
```

Comment:
you are using this exactly class in multiple places, why don't you build a reusable class in the `scss` global folder.
You can use it in the jsx like `text-red` simply without css modules